src/modules/home/components/images-carousel.tsx

```diff
@@ -0,0 +1,99 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Slider from 'react-slick';
+import ArrowBackIosNewIcon from '@mui/icons-material/ArrowBackIosNew';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import { type AccommodationImage } from '../types/accommodation-image.type';
+import { NoDataAvailable } from '#/shared/components/no-data-response';
+
+const styles = {
+  prevButton: {
+    position: 'absolute',
+    top: '50%',
+    left: '-170px',
+    transform: 'translateY(-50%)',
+    color: '#fff',
+    minWidth: '50px',
+    height: '50px',
+    borderRadius: '50%',
+    boxShadow: '0 4px 10px rgba(0,0,0,0.3)',
+    zIndex: 10,
+    '&:hover': { backgroundColor: 'GrayText' },
+  },
+  nextButton: {
```

Comment:
I think this style is the same as previous one, right? Why not keep only one of them with more general name instead of repeating

Reply:
no only one thing is different prevButton has left -170px and nextButton right -170px